@SpriteOvO, Thanks for your PR, this is error log.
Please take a look:
````
Change Dir: '/mnt/vcpkg-ci/buildtrees/sigmatch/x64-linux-dbg'

Run Build Command(s): /mnt/vcpkg-ci/downloads/tools/ninja-1.10.2-linux/ninja -v -v -j33 install
[1/7] /usr/bin/c++  -I/mnt/vcpkg-ci/buildtrees/sigmatch/src/v0.2.0-bf43ea0aed.clean/include -fPIC -g -std=gnu++20 -MD -MT examples/CMakeFiles/03.custom_target.dir/03.custom_target.cpp.o -MF examples/CMakeFiles/03.custom_target.dir/03.custom_target.cpp.o.d -o examples/CMakeFiles/03.custom_target.dir/03.custom_target.cpp.o -c /mnt/vcpkg-ci/buildtrees/sigmatch/src/v0.2.0-bf43ea0aed.clean/examples/03.custom_target.cpp
FAILED: examples/CMakeFiles/03.custom_target.dir/03.custom_target.cpp.o 
/usr/bin/c++  -I/mnt/vcpkg-ci/buildtrees/sigmatch/src/v0.2.0-bf43ea0aed.clean/include -fPIC -g -std=gnu++20 -MD -MT examples/CMakeFiles/03.custom_target.dir/03.custom_target.cpp.o -MF examples/CMakeFiles/03.custom_target.dir/03.custom_target.cpp.o.d -o examples/CMakeFiles/03.custom_target.dir/03.custom_target.cpp.o -c /mnt/vcpkg-ci/buildtrees/sigmatch/src/v0.2.0-bf43ea0aed.clean/examples/03.custom_target.cpp
In file included from /mnt/vcpkg-ci/buildtrees/sigmatch/src/v0.2.0-bf43ea0aed.clean/examples/03.custom_target.cpp:20:
/mnt/vcpkg-ci/buildtrees/sigmatch/src/v0.2.0-bf43ea0aed.clean/include/sigmatch/sigmatch.hpp:82:10: fatal error: format: No such file or directory
   82 | #include <format>
      |          ^~~~~~~~
compilation terminated.
[2/7] /usr/bin/c++  -I/mnt/vcpkg-ci/buildtrees/sigmatch/src/v0.2.0-bf43ea0aed.clean/include -fPIC -g -std=gnu++20 -MD -MT examples/CMakeFiles/02.custom_reader.dir/02.custom_reader.cpp.o -MF examples/CMakeFiles/02.custom_reader.dir/02.custom_reader.cpp.o.d -o examples/CMakeFiles/02.custom_reader.dir/02.custom_reader.cpp.o -c /mnt/vcpkg-ci/buildtrees/sigmatch/src/v0.2.0-bf43ea0aed.clean/examples/02.custom_reader.cpp
FAILED: examples/CMakeFiles/02.custom_reader.dir/02.custom_reader.cpp.o 
/usr/bin/c++  -I/mnt/vcpkg-ci/buildtrees/sigmatch/src/v0.2.0-bf43ea0aed.clean/include -fPIC -g -std=gnu++20 -MD -MT examples/CMakeFiles/02.custom_reader.dir/02.custom_reader.cpp.o -MF examples/CMakeFiles/02.custom_reader.dir/02.custom_reader.cpp.o.d -o examples/CMakeFiles/02.custom_reader.dir/02.custom_reader.cpp.o -c /mnt/vcpkg-ci/buildtrees/sigmatch/src/v0.2.0-bf43ea0aed.clean/examples/02.custom_reader.cpp
In file included from /mnt/vcpkg-ci/buildtrees/sigmatch/src/v0.2.0-bf43ea0aed.clean/examples/02.custom_reader.cpp:19:
/mnt/vcpkg-ci/buildtrees/sigmatch/src/v0.2.0-bf43ea0aed.clean/include/sigmatch/sigmatch.hpp:82:10: fatal error: format: No such file or directory
   82 | #include <format>
      |          ^~~~~~~~
compilation terminated.
[3/7] /usr/bin/c++  -I/mnt/vcpkg-ci/buildtrees/sigmatch/src/v0.2.0-bf43ea0aed.clean/include -fPIC -g -std=gnu++20 -MD -MT examples/CMakeFiles/01.hello_sigmatch.dir/01.hello_sigmatch.cpp.o -MF examples/CMakeFiles/01.hello_sigmatch.dir/01.hello_sigmatch.cpp.o.d -o examples/CMakeFiles/01.hello_sigmatch.dir/01.hello_sigmatch.cpp.o -c /mnt/vcpkg-ci/buildtrees/sigmatch/src/v0.2.0-bf43ea0aed.clean/examples/01.hello_sigmatch.cpp
FAILED: examples/CMakeFiles/01.hello_sigmatch.dir/01.hello_sigmatch.cpp.o 
/usr/bin/c++  -I/mnt/vcpkg-ci/buildtrees/sigmatch/src/v0.2.0-bf43ea0aed.clean/include -fPIC -g -std=gnu++20 -MD -MT examples/CMakeFiles/01.hello_sigmatch.dir/01.hello_sigmatch.cpp.o -MF examples/CMakeFiles/01.hello_sigmatch.dir/01.hello_sigmatch.cpp.o.d -o examples/CMakeFiles/01.hello_sigmatch.dir/01.hello_sigmatch.cpp.o -c /mnt/vcpkg-ci/buildtrees/sigmatch/src/v0.2.0-bf43ea0aed.clean/examples/01.hello_sigmatch.cpp
In file included from /mnt/vcpkg-ci/buildtrees/sigmatch/src/v0.2.0-bf43ea0aed.clean/examples/01.hello_sigmatch.cpp:23:
/mnt/vcpkg-ci/buildtrees/sigmatch/src/v0.2.0-bf43ea0aed.clean/include/sigmatch/sigmatch.hpp:82:10: fatal error: format: No such file or directory
   82 | #include <format>
      |          ^~~~~~~~
compilation terminated.
ninja: build stopped: subcommand failed.

````

--- a/ports/sigmatch/portfile.cmake
+++ b/ports/sigmatch/portfile.cmake
@@ -1,0 +1,22 @@
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO SpriteOvO/sigmatch
+    REF v0.2.0
+    SHA512 a2ae12bf2da4de4b4b65f443febca8bec5ded2cdcbfe5c166538869431558241883576fed04fc373b60fe5b5709c96a56110181d3b1c07dbb42ecfdddae74c06
+    HEAD_REF main
+)
+
+set(VCPKG_BUILD_TYPE release) # header-only
+
+vcpkg_cmake_configure(
+    SOURCE_PATH "${SOURCE_PATH}"
+    OPTIONS
+        -DSIGMATCH_BUILD_TESTS=OFF
+)
+vcpkg_cmake_install()
+
+vcpkg_cmake_config_fixup(CONFIG_PATH lib/cmake/sigmatch)
+
+vcpkg_install_copyright(FILE_LIST "${SOURCE_PATH}/LICENSE")
+
+file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/lib")

--- a/ports/sigmatch/vcpkg.json
+++ b/ports/sigmatch/vcpkg.json
@@ -1,0 +1,18 @@
+{
+  "name": "sigmatch",
+  "version": "0.2.0",
+  "description": "Modern C++ 20 signature match / search library",
+  "homepage": "https://github.com/SpriteOvO/sigmatch",
+  "license": "Apache-2.0",
+  "supports": "windows & !uwp",
+  "dependencies": [
+    {
+      "name": "vcpkg-cmake",
+      "host": true
+    },
+    {
+      "name": "vcpkg-cmake-config",
+      "host": true
+    }
+  ]
+}

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -7628,6 +7628,10 @@
       "baseline": "2.5.1",
       "port-version": 0
     },
+    "sigmatch": {
+      "baseline": "0.2.0",
+      "port-version": 0
+    },
     "signalrclient": {
       "baseline": "1.0.0-beta1-9",
       "port-version": 5

--- a/versions/s-/sigmatch.json
+++ b/versions/s-/sigmatch.json
@@ -1,0 +1,9 @@
+{
+  "versions": [
+    {
+      "git-tree": "83b6f5b1da02efa0313fdf8caa5ee44f41047143",
+      "version": "0.2.0",
+      "port-version": 0
+    }
+  ]
+}

